inferno-framework/tutorial-test-kit
# FI-3166: Step 1 solution upgrade

## Summary
This PR updates the Inferno template version in step-1-solution branch. It includes the solution to step-1 of the tutorial.

## Testing
Run locally with Docker:
- Run `./run.sh` in this repository
- Navigate to `http://localhost`
- Run test kit with the following inputs:
    - FHIR Base Server URL: `https://inferno.healthit.gov/reference-server/r4`
    - access_token: `SAMPLE_TOKEN`
    - Patient ID: `85`
- You should get the following results:
<img width="300" alt="image" src="https://github.com/user-attachments/assets/b47458c9-cce5-4675-b98d-620f16ac1b0e">

Also run with Ruby.
